clang/include/clang/Basic/DiagnosticGroups.td

```diff
@@ -813,19 +813,19 @@ changes to one object won't affect the others, the object's initializer will run
 once per copy, etc.
 
 Specifically, this warning fires when it detects an object which:
-  1. Is defined as ``inline`` in a header file (so it might get compiled into multiple libaries), and
-  2. Has external linkage (otherwise it's supposed to be duplicated), and
-  3. Has hidden visibility.
```

Comment:
I'd keep this file in a separate PR

Reply:
then you can just rebase this one

Reply:
OK. Then I will close https://github.com/llvm/llvm-project/pull/142701